src/e2e-test/features/pipeline.contextmenu.feature

```diff
@@ -0,0 +1,111 @@
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+@Integration_Tests
+Feature: Pipeline Context Menu
+
+  @PIPELINE_CONTEXT_MENU_TEST
+  Scenario: Show plugin level context menu
+    When Open Pipeline Studio Page
+    When Create a simple pipeline
+    Then Open source node context menu
+    Then Validate Copy Plugin and Delete Plugin options are visible
+
+  @PIPELINE_CONTEXT_MENU_TEST
+  Scenario: Ensure functionality by clicking on plugin context menu options
+    When Open Pipeline Studio Page
+    When Create a simple pipeline
+    Then Delete source node through context menu
+    Then Delete transform node through context menu
+    Then Delete sink node through context menu
+    Then Verify export button is disabled
+    Then Undo the deletions
+    Then Export and verify the pipeline has 2 connections and 3 stages
+
+  @PIPELINE_CONTEXT_MENU_TEST
+  Scenario: Show pipeline level icon
+    When Open Pipeline Studio Page
+    When Create a simple pipeline
+    Then Right click on canvas
+    Then Ensure canvas context menu option "Wrangle" is displayed
+    Then Ensure canvas context menu option "Zoom In" is displayed
+    Then Ensure canvas context menu option "Zoom Out" is displayed
+    Then Ensure canvas context menu option "Fit To Screen" is displayed
+    Then Ensure canvas context menu option "Align" is displayed
+    Then Ensure canvas context menu option "Paste" is displayed
+
+
+
+  @PIPELINE_CONTEXT_MENU_TEST
+  Scenario: Ensure clicking zoom in and zoom out works properly
+    When Open Pipeline Studio Page
+    When Create a simple pipeline
+    Then Verify "default" zoom level
+    Then Right click on canvas
+    Then Click on "Zoom In" from canvas context menu option
+    Then Verify "first" zoom level
+    Then Right click on canvas
+    Then Click on "Zoom In" from canvas context menu option
+    Then Verify "second" zoom level
+    Then Right click on canvas
+    Then Click on "Zoom Out" from canvas context menu option
+    Then Verify "first" zoom level
+    Then Right click on canvas
+    Then Click on "Zoom Out" from canvas context menu option
+    Then Verify "default" zoom level
+
+  @PIPELINE_CONTEXT_MENU_TEST
+  Scenario: Ensure clicking fit to screen works properly
+    When Open Pipeline Studio Page
+    When Create a complex pipeline
+    Then Verify complex pipeline nodes are visible
+    Then Zoom in ten times on pipeline canvas
+    Then Verify complex pipeline nodes are not visible
+    Then Right click on canvas
+    Then Click on "Fit To Screen" from canvas context menu option
+    Then Verify complex pipeline nodes are visible
+
+  @PIPELINE_CONTEXT_MENU_TEST @ignore
```

Comment:
The last three scenarios have `@ignore` tags since I haven't figured out a way to give clipboard access to chrome while running in `headless` mode. 
I found a solution on stackoverflow that works well when `headless` mode is disabled, and all these three scenarios passed locally. However, the solution adds a preference to the chrome driver for giving access to clipboard. Preferences are not supported in `headless` mode.
You can read more on the solution and issue here:
[Solution 1](https://stackoverflow.com/questions/53669639/enable-clipboard-in-automated-tests-with-protractor-and-webdriver), [Solution 2](https://testingbot.com/support/selenium/permission-popups#:~:text=Clipboard%20Permissions), [preferences not supported in headless](https://bugs.chromium.org/p/chromedriver/issues/detail?id=1925)

Reply:
I'll hold off on merging for another day, but the fix will most likely be on the chrome driver in `e2e-tests` repo. These changes will stay, I'll just remove the @ignore tags once the issue is resolved.